app.cr

```diff
@@ -27,24 +27,24 @@ def filter(repos, filter)
 end
 
 def fetch_sort(context)
-  sort = context.params["sort"]? || "stars"
+  sort = context.params["sort"]?.try(&.to_s) || "stars"
   sort = "stars" unless SORT_OPTIONS.includes?(sort)
   sort
 end
 
-def fetch_filter(context)
-  context.params["filter"]?.try(&.strip.downcase) || ""
+def fetch_filter(env)
+  env.params["filter"]?.try(&.to_s.strip.downcase) || ""
 end
 
-private def matches_filter?(item: GithubRepo, filter: String)
+private def matches_filter?(item : GithubRepo, filter : String)
```

Comment:
This seems like a stylistic change? Is there any other reason for the extra space?